src/chrome/content/rules/PConline.xml

```diff
@@ -18,13 +21,12 @@
 	<target host="img0.pconline.com.cn" />
 		<test url="http://img0.pconline.com.cn/pconline/1408/11/5250899_IMG_9970.jpg" />
 	<target host="img1.pconline.com.cn" />
-		<test url="http://img1.pconline.com.cn/piclib/200806/25/batch/1/4732/12144096030210g4cndrspy.jpg" />
+		<test url="http://img1.pconline.com.cn/pconline/0807/25/1364488_IMG_94072.jpg" />
 	<target host="img2.pconline.com.cn" />
-		<test url="http://img2.pconline.com.cn/pconline/0707/25/1065484_2.jpg" />
+		<test url="http://img2.pconline.com.cn/pconline/0908/17/1742090_090817_hero_001s.jpg" />
 	<target host="img3.pconline.com.cn" />
 		<test url="http://img3.pconline.com.cn/pconline/0611/20/908527_20061120samsung931bw_1s.jpg" />
 	<target host="img-arch.pconline.com.cn" />
-	<target host="img0-arch.pconline.com.cn" />
```

Comment:
Custom test urls WFM

Reply:
https://travis-ci.org/EFForg/https-everywhere/jobs/342766717#L641

Reply:
https://travis-ci.org/EFForg/https-everywhere/jobs/342766717#L641

Reply:
Try with an exclusion on `$` and a custom test url, e.g. `http://img0-arch.pconline.com.cn/pcauto/1107/21/1575352_pic.png`